documentation/docs/examples/point-to-point-measurement.mdx

```diff
@@ -0,0 +1,211 @@
+---
+id: point-to-point-measurement
+title: Point to point measurement
+description: Demo shows how to measure point to point distance in Reveal.
+---
+
+import { DemoWrapper } from '@site/docs/components/DemoWrapper';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The `MeasurementTool` is a measuring tool for point to point distance in 3D models, by clicking to- and from-points. The default measurement unit is meters, but the tool provides callbacks to allow configuring the unit.
+When exiting from the tool, all the measurement will be removed from the `Cognite3DViewer`.
+
+To effectively edit measuring line properties, tool is configurable with `MeasurementOptions` options to set
+- Custom label content for each measurement.
+- Size of line width in meters.
+- Color of line.
+
+:::note
+Recommended: `CameraControlsOptions.changeCameraTargetOnClick` should be set to `false` to avoid click event registering for camera when `MeasurementTool` is active. Also avoid any custom click-handler that might interfere with the measurement tool.
+:::
+
+<DemoWrapper name="Cognite3DViewerDemo" />
+
+## Point to point distance measurment
+To enter point-to-point measurement mode, use `MeasurementTool.enterMeasurementMode()`. This will register events for handling mouse clicks and touch events to allow placing the measurement end-points.
```

Comment:
```suggestion
To enter point-to-point measurement mode, use `MeasurementTool.enterMeasurementMode()`. This will register events for handling mouse clicks and touch events to allow placing the measurement end-points. Note that the tool is optimized for use with mouse.

To exit measurement mode, call `exitMeasurementMode()`. The tool will also exit measurement mode if user presses Escape key.
```